lib/experimental/new_embed.dart

```diff
@@ -134,7 +135,7 @@ class NewEmbed {
     userCodeEditor.readOnly = value;
     executeButton.disabled = value;
     formatButton.disabled = value;
-    reloadGistButton.disabled = value || gistId.isEmpty;
+    reloadGistButton.disabled = value;
```

Comment:
What happens here if they come in without a gist ID and without any injected code? Do they just get an empty editor?

Reply:
Yep, it currently clears the editor. I just realized there's an issue with the material design buttons not appearing disabled when they are set to disabled: https://github.com/dart-lang/dart-pad/issues/1196

Reply:
I'll fix that issue in a separate PR.